webxrlayers-1.bs

```diff
@@ -1386,6 +1386,54 @@ when the [=XR Compositor=] can no longer reproject the layer.
 
 The author SHOULD redraw the content of the layer at the next [=XR animation frame=]. The event must be of type {{XRLayerEvent}}.
 
+Hit testing {#hittesting}
+===========
+
+XRLayerHitResult {#xrlayerhitresulttype}
+----------------
+
+<pre class="idl">
+interface XRLayerHitResult {
+  XRCompositionLayer layer;
+  unsigned long x;
+  unsigned long y;
+  DOMPointInit position;
+};
+</pre>
+
+The <dfn dict-member for="XRLayerHitResult">layer</dfn> member defines what layer was hit.
+
+The <dfn dict-member for="XRLayerHitResult">x</dfn> and <dfn dict-member for="XRLayerHitResult">y</dfn> member define the x and y position
```

Comment:
In what coordinate space?

Reply:
The one of the XRLayer. It has a pixel width/height

Reply:
Alright, that should be specified :smile:

Reply:
I changed it to return a value between 0 and 1. I forgot about video layers :-)